core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,28 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+.popover{
+  font-size: 13px;
+  max-width: 300px;
+  text-align: center;
+  z-index: -1;
+  color: white;
+  background-color: black;
+}
+.popover.bottom > .arrow{
+  top: -11px;
+  left: 50%;
+  margin-left: -11px;
```

Comment:
@shivanXI A quick note for clean merging. The thirdparty.css already brings in the code other than color properties. 

Example: Below properties already comes from thirdparty.css
```
top: -11px;
left: 50%;
margin-left: -11px;
```
In my view, 
Adding it again would be just duplicating the properties from thirdparty.css in this file. (The same  comment with  other classes `.popover class`, `.popover.bottom > .arrow:after` and `.popover.bottom > .arrow`)
As this PR focuses on "changing color combination", just color based properties should be pushed.

@kevinlee12  and @seanlip, Do suggest your opinion on this? Will it (duplicating ) affect the speed or code somehow?